modules/ROOT/pages/api-groups-landing-page.adoc

```diff
@@ -0,0 +1,69 @@
+= API Groups
```

Comment:
```suggestion
= Advantages of Using API Groups
```

Reply:
Hmm...this page is that and above so will keep as is for now.